mappings/net/minecraft/client/particle/ParticleAnimated.mapping

```diff
@@ -1,14 +1,20 @@
-CLASS bnr
+CLASS bnr net/minecraft/client/particle/ParticleAnimated
 	FIELD M targetColorRed F
 	FIELD N targetColorGreen F
 	FIELD O targetColorBlue F
 	FIELD P changesColor Z
+	FIELD a textureId I
+	FIELD b frameCount I
 	METHOD <init> (Laiw;DDDIIF)V
 		ARG 0 world
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+		ARG 4 textureId
+		ARG 5 frameCount
 	METHOD a update ()V
+	METHOD a getColorMultiplier (F)I
+	METHOD c hasTransparency ()Z
```

Comment:
`isTransparent` may be better if transparency is a binary thing, `hasTransparency` sounds like transparency could be a value between 0 and 1.